app/src/main/java/org/fossasia/openevent/app/common/app/glide/GlideAPI.java

```diff
@@ -0,0 +1,35 @@
+package org.fossasia.openevent.app.common.app.glide;
+
+import android.content.Context;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.GlideBuilder;
+import com.bumptech.glide.Registry;
+import com.bumptech.glide.annotation.GlideModule;
+import com.bumptech.glide.integration.okhttp3.OkHttpUrlLoader;
+import com.bumptech.glide.load.engine.cache.InternalCacheDiskCacheFactory;
+import com.bumptech.glide.load.model.GlideUrl;
+import com.bumptech.glide.module.AppGlideModule;
+
+import java.io.InputStream;
+
+@GlideModule
+public final class GlideAPI extends AppGlideModule {
+
+    @Override
+    public void registerComponents(Context context, Glide glide, Registry registry) {
+        registry.replace(GlideUrl.class, InputStream.class, new OkHttpUrlLoader.Factory());
```

Comment:
This isn't needed if we are not using logger. Please remove this and add a comment to use logger in the future